system/include/emscripten/websocket.h

```diff
@@ -1,3 +1,10 @@
+/*
+ * Copyright 2018 The Emscripten Authors.  All rights reserved.
```

Comment:
2019?

Reply:
I'm using the year of file creation here.